### PR TITLE
Pullrequest house keeping 2023/06/05

### PR DESCRIPTION
**Pull request by BenSartor**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -69,8 +68,8 @@ dependencies {
     //noinspection VulnerableLibrariesLocal
     testImplementation("com.h2database:h2:1.4.200")
     testImplementation("org.springframework.boot:spring-boot-starter-test")
-    testImplementation("org.mockito:mockito-core:${mockitoVersion}")
-    testImplementation("org.mockito:mockito-inline:${mockitoVersion}")
+    testImplementation("org.mockito:mockito-core:5.3.1")
+    testImplementation("org.mockito:mockito-inline:5.2.0")
```
[the remark this answers is not in the document]
Comment (jritzerfeld) [reply build.gradle]: I thought more about removing the specific version while keeping the dependencies as such...

[the remark this answers is not in the document]
Comment (BenSartor) [reply build.gradle]: Done

[the remark this answers is not in the document]
Comment (jritzerfeld) [reply build.gradle]: I'm not sure whether we can really remove it after updating to Spring Boot 3 but we should definitively try it.

**Review by jritzerfeld — APPROVED** [review]
Excellent!

**Pull request merged**